src/appleseed/renderer/modeling/input/inputbinder.cpp

```diff
@@ -682,6 +687,16 @@ bool InputBinder::try_bind_assembly_entity_to_input(
                 param_value,
                 input);
             return true;
+
+          case SymbolTable::SymbolSourceEntity:
+              bind_max_texture_to_input(
```

Comment:
`switch` statements are formatted like the rest of the code, using four-space indentations, i.e. case statements' _bodies_ are indented relatively to `switch` with four spaces. Ony `case` statements themselves are positioned halfway, i.e. are indented with two spaces only.